hack/verify-godeps.sh

```diff
@@ -56,8 +56,11 @@ trap cleanup EXIT
 # build the godep tool
 export GOPATH="${_tmpdir}"
 go get -u github.com/tools/godep 2>/dev/null
-go install github.com/tools/godep 2>/dev/null
 GODEP="${_tmpdir}/bin/godep"
+pushd "${GOPATH}/src/github.com/tools/godep" > /dev/null
+  git checkout v53
+  "${GODEP}" go install
```

Comment:
It uses the latest and greatest to build the old version. If they can't even make godep build itself, they have serious problems...   works today, and hopefully we can revert this entirely shortly-ish